model/repositories.go

```diff
@@ -86,8 +84,8 @@ func (e *EventRepositoryGorm) Persist(ctxt context.Context, entity AggregateInte
 		if event.Meta.Group == "" {
 			event.Meta.Group = e.GroupID
 		}
-		if event.Meta.SchemaName == "" {
-			event.Meta.SchemaName = schemaName
+		if event.Meta.EntityType == "ContentEntity" || event.Meta.EntityType == "" {
```

Comment:
would entity types be "ContentEntity" still? (it shouldn't be). Maybe find the source from where "ContentEntity" is set and fix it there (instead of sorting it out in persist